libs/shared/src/components/CarrierBox.vue

```diff
@@ -18,13 +18,14 @@
 
 <script lang="ts" setup>
 import {toRefs} from 'vue';
-import {type CarrierName} from '@myparcel/constants';
+import {type CarrierIdentifier} from '@myparcel-do/shared';
```

Comment:
dit kan niet, dit moet een relatief pad zijn want je zit al in shared

Reply:
Ik heb het gewijzigd